python/src/viewer.py

```diff
@@ -758,19 +900,39 @@ def PlotDesign(design, bounds, Nx, Ny, figure=None, axes=[], plot_kwargs={}, axe
         return figure
 
     # General case
+
+    # For the diagonal of a multidimensional plot, 
+    # disable the ticks, always
+    diagonal_axes_kwargs = axes_kwargs.copy() # An independent copy
```

Comment:
why using a `copy`?

Reply:
I think that this prevents from modifying the user input argument, isn'it?

Reply:
but graphs are done just after, so probably it is useless